Content.Server/Radiation/Components/GeigerComponent.cs

```diff
@@ -0,0 +1,42 @@
+using Content.Shared.Radiation.Components;
+using Robust.Shared.Audio;
+
+namespace Content.Server.Radiation.Components;
+
+/// <inheritdoc/>
+[RegisterComponent]
+[ComponentReference(typeof(SharedGeigerComponent))]
+public sealed class GeigerComponent : SharedGeigerComponent
+{
+    /// <summary>
+    ///     Map of sounds that should be play on loop for different radiation levels.
+    /// </summary>
+    [DataField("sounds")]
+    public Dictionary<GeigerDangerLevel, SoundSpecifier> Sounds = new()
+    {
+        {GeigerDangerLevel.Low, new SoundPathSpecifier("/Audio/Items/Geiger/low.ogg")},
+        {GeigerDangerLevel.Med, new SoundPathSpecifier("/Audio/Items/Geiger/med.ogg")},
+        {GeigerDangerLevel.High, new SoundPathSpecifier("/Audio/Items/Geiger/high.ogg")},
+        {GeigerDangerLevel.Extreme, new SoundPathSpecifier("/Audio/Items/Geiger/ext.ogg")}
+    };
+
+    /// <summary>
+    ///     If true it will be active only when player equipped it.
+    /// </summary>
+    [DataField("attachedToSuit")]
+    public bool AttachedToSuit;
```

Comment:
These could be on shared? Most code should be on shared by default unless there's solid reason not to (e.g. radiation which is expensive).

Reply:
I don't mind to move this in shared component, but you understand that it will be shown in VV on a client side? I mean, what the point if it's only used server side?

Reply:
All of the audio handling is on server when client could do it and predict it.